### PR TITLE
bump RET and Scrypto to version 

### PR DESCRIPTION
**Pull request by CyonAlexRDX**
Without the typo `PUSH_TO_AUTHZONE -> PUSH_TO_AUTH_ZONE`

> [!NOTE]
> Omar is "fixing a release" tag for us to use instead, but this seems to work...
> When he pings us with a tuple of git tags for RET and Scrypto we should bump to them
> because the commit https://github.com/radixdlt/radixdlt-scrypto/commit/27da7a08fac63d36bd1ab209840fb014f470ebc6 has warning
> "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

### CODE REVIEW

**Review by matiasbzurovski — COMMENTED** [review]
Thanks for taking care of this!

Think we should update UniFFI crate [dependency](https://github.com/radixdlt/sargon/blob/main/crates/sargon-uniffi/Cargo.toml#L24) to RET as well?

**Comment by CyonAlexRDX** [comment]
> Thanks for taking care of this!
> 
> Think we should update UniFFI crate [dependency](https://github.com/radixdlt/sargon/blob/main/crates/sargon-uniffi/Cargo.toml#L24) to RET as well?

thanks, fixing now

**Comment by CyonAlexRDX** [comment]
I probably need to update some things in the `sargon-uniffi` crate... fixing now!

File: crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs
Comment (matiasbzurovski) [inline crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs:219]: why has this changed?

File: crates/sargon/Cargo.toml
Comment (GhenadieVP) [inline crates/sargon/Cargo.toml:77]: Can we have constants in Cargo.toml? It would be really nice to have to change this value only once. Not blocking this PR, just wondering

File: crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs
Comment (GhenadieVP) [reply crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs:219]: AccountUpdateSettings is no longer a reserved instruction

**Review by GhenadieVP — APPROVED** [review]

File: crates/sargon/Cargo.toml
Comment (CyonAlexRDX) [reply crates/sargon/Cargo.toml:77]: Nope not possible I think. I've been wanting it for a loooong time.

File: crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_intents.rs
Comment (CyonAlexRDX) [inline crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_intents.rs:11]: @matiasbzurovski we should rename this file to `
child_subintent_specifiers` , I leave that to you

File: crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_subintent.rs
Comment (CyonAlexRDX) [inline crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_subintent.rs:4]: we should rename this file, I leave that to you @matiasbzurovski

**Pull request merged**
